jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -112,6 +118,13 @@ def _create_viewer_callbacks(self, viewer):
     def _on_viewer_added(self, msg):
         self._create_viewer_callbacks(self.app.get_viewer_by_id(msg.viewer_id))
 
+    def _on_global_display_unit_changed(self, msg):
+        new_unit = u.Unit(msg.unit)
+        uc = self.app._jdaviz_helper.plugins.get("Unit Conversion", None)
+        if self.config == "cubeviz":
+            if new_unit in uc.sb_unit.choices:
```

Comment:
can we check `msg.axis` instead of relying on accessing the other plugin?

Reply:
Hello Kyle, I need to extract the unit lists from the sb dropdown. The `msg.axis` always returns "flux," but I specifically need the sb list of choices in my case. How can I differentiate between flux and sb in these scenarios?